src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Services/ContentApiClient.cs

```diff
@@ -57,6 +57,40 @@ public ContentApiClient(ILogger<ContentApiClient> logger, HttpClient httpClient)
             await response.Content.ReadFromJsonAsync<PaginatedListViewModel<PublicationSearchResultViewModel>>();
 
         return publications
-            ?? throw new Exception("Could deserialize publications from content API.");
+            ?? throw new Exception("Could not deserialize publications from content API.");
+    }
+
+    public async Task<Either<ActionResult, PublishedPublicationSummaryViewModel>> GetPublication(Guid publicationId)
+    {
+        var response = await _httpClient.GetAsync($"api/publications/{publicationId}/summary");
+
+        if (!response.IsSuccessStatusCode)
+        {
+            var message = await response.Content.ReadAsStringAsync();
+            _logger.LogError(StringExtensions.TrimIndent(
+                $"""
+                 Failed to retrieve publication '{publicationId}' with status code: {response.StatusCode}. Message:
+                 {message}
+                 """));
+
+            switch (response.StatusCode)
+            {
+                case HttpStatusCode.BadRequest:
+                    return new BadRequestObjectResult(await response.Content.ReadFromJsonAsync<ValidationProblemViewModel>());
+                case HttpStatusCode.NotFound:
+                    bool hasContent = response.Content.Headers.ContentLength > 0;
+                    return hasContent
+                        ? new NotFoundObjectResult(await response.Content.ReadFromJsonAsync<ProblemDetails>())
+                        : new NotFoundResult();
```

Comment:
It's not necessary to pipe through the response from the content API. As discussed in person, this endpoint is pretty simple so there's no ambiguity about what the 404 relates to.

Think we can simplify this to just return a `NotFoundResult`

Reply:
done :)